emoji/src/main/java/com/vanniktech/emoji/EmojiUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.vanniktech.emoji;
+
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class EmojiUtils {
```

Comment:
Let's leave it here for now (in EmojiUtils) I anyways thought of restructuring it a bit before the 0.5.0 release.